src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -99,6 +102,20 @@ function getFacilitiesInfo() {
   $('#btnCloseAllFacilities').hide();
 }
 
+function sortFacility(facility) {
+  facility.sort((facilityA, facilityB) => {
+    const NameA = facilityA.facilityName.toUpperCase();
+    const NameB = facilityB.facilityName.toUpperCase();
```

Comment:
Are these supposed to start with capital letters? Variables shouldn't start with capital letters right?

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
Fixed